config/Version.public.xcconfig

```diff
@@ -1,4 +1,4 @@
 VERSION_SHORT=20.4
 
 // Public long version example: VERSION_LONG=9.9.0.0
-VERSION_LONG=20.4.0.0
+VERSION_LONG=20.4.0.1
```

Comment:
fwiw I kinda find it more confusing that you made that `release/20.4.0.1` branch to not include the two PRs you missed… given that in `release/20.4` I see that you also bumped the version number (to `20.4.0.2`) now…

So am I reviewing an already-obsolete PR that got superseded by a new beta `20.4.0.2` you made more recently? Or was that bump in `release/20.4` in anticipation of a future release? Or some glitch with the tooling?

Maybe keeping the PR being from `release/20.4` and mentioning in the PR description that those were not included would have made things easier? Especially since that's usually how we do it (especially it's common to do a beta, make the release PR, and have a new late betafix PR land in the release branch in-between when that beta was done and the corresponding PR got merged… in which cases the late betafix PR was not present by the time you created the release PR description… but got added to the branch later — that happened to me once or twice in the past)